Gold-Dataset-EDA.ipynb

```diff
@@ -0,0 +1,1584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plot_timeseries(df,'Price', 22) # for one month 
</pre><p>I would prefer to have the 22 or the 260 in const variables in order to make the code readable without the need of the comment</p><p>e.g: WORK_DAYS_PER_MONTH = 22</p><p>	   WORK_DAYS_PER_YEAR = 260</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/shamelj/Precious-Metals-Forecaster/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>